backend/workflow_manager/workflow/models/file_history.py

```diff
@@ -39,6 +39,7 @@ def is_completed(self) -> bool:
         db_comment="Error message",
     )
     result = models.TextField(blank=True, db_comment="Result from execution")
+    meta_data = models.TextField(blank=True, db_comment="MetaData from execution")
```

Comment:
NIT: Consider treating it as a single word `metadata`. I see an inconsistency with how we use snake / camel case and it'll be good to remain consistent

Reply:
The reason is the user has more flexible control of enabling the highlighting option. 
Enabling this setting adapter level will have an effect in all the places where we use the adapter. 
Now we have a control at per tool level.